index.html

```diff
@@ -81,40 +81,12 @@ <h3>Closure</h3>
       </li>
     </ul>
     <a href="#" class="see-program lato">SEE THE WHOLE PROGRAM</a>
-    <a href="#" id="join-btn" class="lato">Join Atlantis World Cup 2022</a>
+    <a href="#" id="join-btn" class="lato">Join Dankira Cup 2022</a>
   </section>
-  <div class="hor-list">
-    <div class="hor-ins">
-        <div class="horse-card" id="horse-card-skeleton1">
-            <picture>
-          <img src="media/horse-1.jpg" alt="horse1">
-            </picture>
-        </div>
-        <div class="lato">
-          <h3>heading placeholder</h3>
-          <h5>highlights placeholder</h5>
-          <hr>
-          <h6>description placeholder</h6>
-        </div>
-    </div>
-    <div class="hor-ins1">
-        <div class="horse-card" id="horse-card-skeleton2">
-            <picture>
-              <img src="media/horse-2.jpg" alt="horse2">
-            </picture>
-            <div class="lato">
-              <h3>heading placeholder</h3>
-              <h5>highlights placeholder</h5>
-              <hr>
-              <h6>description placeholder</h6>
-            </div>
-        </div>
-    </div>
-   </div>
   <section id="featured-horses">
-    <h2 class="lato">Featured Horses <span></span></h2>
+    <h2 class="lato">Featured Cars <span></span></h2>
```

Comment:
Great use of semantics 😎 The proper use of tags is key to having better HTML! Well done using section and footer tags 💪 We must not forget to use the `article` tag! Kindly make the necessary changes in your speaker's HTML dynamic creation so that each of them has an `article` tag 🤓